parachain/pallets/verifier/src/lib.rs

```diff
@@ -0,0 +1,87 @@
+#![allow(unused_variables)]
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use frame_system as system;
+use frame_support::{decl_module, decl_storage, decl_event, decl_error,
+	dispatch::DispatchResult};
+use sp_runtime::traits::Hash;
+use sp_std::prelude::*;
+
+use artemis_core::{AppId, Message, Verifier, VerificationInput};
+
+#[cfg(test)]
+mod mock;
+
+#[cfg(test)]
+mod tests;
+
+pub trait Trait: system::Trait {
+	type Event: From<Event> + Into<<Self as system::Trait>::Event>;
+}
+
+decl_storage! {
+	trait Store for Module<T: Trait> as VerifierModule {
+		RelayKey get(fn key) config(): T::AccountId;
```

Comment:
Please add a comment describing RelayKey. Thanks!

Reply:
Good point, is a bit obscure. Will add some more commentary.

Reply:
I've added a bunch of documentation across the parachain in the latest upload.